tests/test_fft.py

```diff
@@ -1,20 +1,62 @@
+"""
+In these tests, we iterate through some valid fft backends in for [common.fft] attribute in config.ini
+using the `config_override` mechanism for `Config` objects, and check for valid results.
+
+Because the determination of which fft backend to use happens at import time, this is somewhat tricky to do.
+Here we employ the `importlib.reload` function while we're iterating through these fft backends to explicitly
+trigger a reload.
+"""
+
+from importlib import reload
 from unittest import TestCase
 
-import numpy as np
+import aspire.utils.numeric
+from aspire.utils.config import config_override
 
-from aspire.utils.numeric import xp
+fft_backends = ('scipy', 'pyfftw')
 
 
 class ConfigTest(TestCase):
-    def setUp(self):
-        pass
+    def testFft(self):
+        for fft_backend in fft_backends:
+            with config_override({'common.fft': fft_backend}):
+                reload(aspire.utils.numeric)
+                xp, fft = aspire.utils.numeric.xp, aspire.utils.numeric.fft
```

Comment:
Why does `xp` have to be used here? Shouldn't that be tested separately?

Reply:
The xp was included because we want the test file works for cupy too, which  can be enabled from config.ini.

Reply:
I don't understand. If the FFT is CuPy, does that require that the numeric module is CuPy as well? If not, we should separate the tests.

Reply:
The numeric module xp is not necessary for using Cupy FFT. But it is necessary to use other functions such as xp.random.random, xp.allclose. The unit tests are rewritten to include Cupy tests starting from commit 49c5bf0.